src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -129,6 +129,20 @@ public void addKnownAllergy() {
 		addOrEditAllergyPage.addAllergyNote("The effect is severe");
 	}
 
+	@When("the user clicks Add Known Allergy button")
+	public void addAllergy() {
+		addOrEditAllergyPage = allergyPage.clickOnAddNewAllergy();
+	}
+
+	@And("the user selects an allergy")
```

Comment:
Change the import statemement package  to import from https://github.com/openmrs/openmrs-contrib-qaframework/tree/master/src/test/java/org/openmrs/contrib/qaframework/page    to cater for the current changes

Reply:
> Change the import statemement package to import from https://github.com/openmrs/openmrs-contrib-qaframework/tree/master/src/test/java/org/openmrs/contrib/qaframework/page to cater for the current changes

@sherrif10 could you shade more light I din't pick well on this